Dockerfile

```
@@ -1,4 +1,5 @@
 FROM jenkins/jenkins:2.414.2-jdk17
 
+COPY logos $JENKINS_HOME/userContent/logos
```

```suggestion
COPY logos /usr/share/jenkins/ref/userContent/logos
```
Ref: https://github.com/jenkinsci/docker#installing-more-tools